README.md

```diff
@@ -0,0 +1,32 @@
+build
+------
+./gradlew fatJar
+
+
+usage
+----------
+* copy build/libs/zookeeper-ldap-roles-all.jar to ZOOKEEPER_HOME/lib directory
+
+* create a configuration properties with following properties
+
+   treadmill_ldap=ldap://10.10.10.10:22389
+   treadmill_ldap_suffix=dc=suffix
+   treadmill_cell=local
+   base_dn_for_admin_role=ou=cells,ou=treadmill,{0}
+   filter_for_admin_role=(&(objectClass=tmCell)(cell={0}))
```

Comment:
just curios - it seems that it does not contain master name in the filter (and indeed i see that you iterate over all results to see if it matches.

nit - we should probably make query faster by matching on master hostname. can be next PR.